tests/unit/test-util/test.submit-addon.js

```diff
@@ -8,8 +8,8 @@ import { assert, expect } from 'chai';
 import JSZip from 'jszip';
 import { afterEach, before, beforeEach, describe, it } from 'mocha';
 import * as sinon from 'sinon';
-// eslint-disable-next-line no-shadow
-import { File, FormData, Response } from 'node-fetch';
+// eslint-disable-next-line no-shadow -- Testing sub-100 Responses, impossible with native
+import { File, Response } from 'node-fetch';
```

Comment:
The tests would fail when the mock `Response` was instantiated with <100 responses instead of throwing in the client. So I left node-fetch as a dev dependency.

Reply:
_me when I paste code from rpl-GPT and everything is still green_

![mother-of-god-omg](https://github.com/user-attachments/assets/ef5d310f-4bf9-4356-9a14-0853369e2277)